conda-recipes/faiss-gpu-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.6.2" %}
+
+package:
+  name: faiss-gpu
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebookresearch/faiss/archive/v{{ version }}.tar.gz
+  sha256: 8be8fcb943e94a93fb0796cad02a991432c0d912d8ae946f4beb5a8a9c5d4932
+  patches:
+    - 0001-add-tolerance-for-ppc64le.patch
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - setuptools
+    - swig
+
+  host:
+    - python
+    - numpy 1.17.*
+    - libopenblas # [ppc64le]
+    - mkl >=2018  # [x86_64]
+    - cudatoolkit ==10.2.*
+    - cudatoolkit-dev ==10.2.*
+  run:
+    - python
+    - numpy 1.17.*
+    - libopenblas # [ppc64le]
+    - mkl >=2018 # [x86_64]
+    - blas=*=mkl # [x64_64]
+    - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
+
+build:
+  number: {{ build }}
```

Comment:
I thought build had to be defined at the top of the file. Does it not?

Reply:
Yep, this will cause a build fail. :)  I looked without enough attention at the `pycuda` recipe and assumed it was some Travis magic. Another commit coming.